contracts/governance/IFeeSharingCollectorProxy.sol

```diff
@@ -1,10 +1,10 @@
 pragma solidity ^0.5.17;
 
 /**
- * @title Interface for contract governance/FeeSharingProxy/FeeSharingProxy.sol
+ * @title Interface for contract governance/FeeSharingCollectorProxy/FeeSharingCollectorProxy.sol
  * @dev Interfaces are used to cast a contract address into a callable instance.
  * */
-interface IFeeSharingProxy {
+interface IFeeSharingCollectorProxy {
```

Comment:
IFeeSharingCollector ?

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/482/commits/e4758f4aa1e7376eafb188b06c4a85702f370842)